data/transactions/logic/doc.go

```diff
@@ -193,6 +195,12 @@ var opDocByName = map[string]string{
 
 	"vrf_verify": "Verify the proof B of message A against pubkey C. Returns vrf output and verification flag.",
 	"block":      "field F of block A. Fail unless A falls between txn.LastValid-1002 and txn.FirstValid (exclusive)",
+
+	"proto":      "Prepare top call frame for a retsub that will assume A args and R return values.",
+	"frame_dig":  "Nth (signed) value from the frame pointer.",
+	"frame_bury": "Replace the Nth (signed) value from the frame pointer in the stack",
+	"pushn":      "Push N 0s onto the stack",
```

Comment:
^ `pushn` somewhat suggests push some arg to stack n times, would be clear if the opcode suggests what is pushed, namely zero in this case. I can also see the need to keep `push` in naming tho, for the stack height is incrementing, so maybe `pushn_zero`?

Reply:
alternatively I can also see maybe we can make this even a pseudo opcode tho, for `pushn` semantics now is equivalent to `N; int 0; dupn`

Reply:
I do like this, in one form or another.  `zeroes` seems good.

Reminder: we have to decide if `pushn`/`zeroes` is even useful, since we have `dupn`. I'm leaning away at this point.

Reply:
Taking numpy as the "definitive source" it's [zeros](https://numpy.org/doc/stable/reference/generated/numpy.zeros.html)